pa_config/rca_master.conf

```diff
@@ -102,9 +102,19 @@
   "muted-deciders": [],
   "muted-actions": [],
 
+  // Default Values are added in the config specific file as well
+  // Modify them as well whenever changing the default values (DeciderConfig.java)
   "decider-config-settings": {
+    // Decreasing order of priority for the type of workload we can expect on the cluster.
+    "workload-type": {
+      "priority-order": ["ingest", "search"]
+    },
+    // Decreasing order of priority for the type of cache which is expected to be consumed more.
+    "cache-type": {
+      "priority-order": ["read-heavy", "write-heavy"]
+    },
     // cache decider - Needs to be updated as per the performance test results
-    "cache-decider-config": {
+    "cache-bounds": {
```

Comment:
We should move default values to code and pick overrides from `rca.conf`. This can be a separate PR/task.

Reply:
This is how it is happening currently. We read the overrides only from the `rca.conf` and default values are hard coded. Currently default and overridden values are the same.